### PR TITLE
AOM-76: Add component to display an error when a module fails to start

### PR DESCRIPTION
**Pull request by mkiterian**
## JIRA TICKET NAME:
[AOM-76 Add component to display an error when a module fails to start](https://issues.openmrs.org/browse/AOM-76)

### SUMMARY:
When the start action on a module fails the user should receive a notification about the failure as well as detailed information about reasons for the failure

### CODE REVIEW

**Comment by brayoh** [comment]
LGTM.

**Review by tduyile04 — COMMENTED** [review]
Looks good. Great job

**Comment by Annettesunday** [comment]
Looks good to me.

**Comment by joel-ace** [comment]
Looks Good

**Comment by Farhanisto** [comment]
Looks good

File: app/js/components/manageApps/StartErrorModal.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          >
+            Mark as Read
+          </button>
+        </ModalFooter>
+      </Modal>
+    );
+  }
+}
+
+StartErrorModal.propTypes = {
```
Comment (andela-dbamidele) [inline app/js/components/manageApps/StartErrorModal.jsx:46]: `React.PropTypes` is deprecated. You should install `prop-types` as a separate dependency.

**Review by andela-dbamidele — COMMENTED** [review]
Awesome job. 👍

File: app/js/components/manageApps/StartErrorModal.jsx (hunk 1 above)
Comment (mkiterian) [reply app/js/components/manageApps/StartErrorModal.jsx:46]: 👍

**Review by LWanjiru — COMMENTED** [review]
Looks good!

**Pull request merged**